### PR TITLE
CI: Add automatic compress action

### PR DESCRIPTION
**Pull request by wolf-yuan-6115**
## 起飛前確認事項

- [ ] 我已詳細確認編輯之文件有符合 [CONTRIBUTING.md](https://github.com/yeecord/docs/blob/master/CONTRIBUTING.md) 文件規範

### CODE REVIEW

**Comment by wolf-yuan-6115** [comment]
It will run every day and open a pr when needed

File: .github/workflows/image-compress.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        id: calibre
+        uses: calibreapp/image-actions@main
+        with:
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          compressOnly: true
+      - name: Create New Pull Request If Needed
+        if: steps.calibre.outputs.markdown != ''
+        uses: peter-evans/create-pull-request@v3
+        with:
+          title: "image: compress images"
```
Comment (Gary50613) [inline .github/workflows/image-compress.yml:25]: uppercase `I`

**Review by Gary50613 — CHANGES_REQUESTED** [review]

File: .github/workflows/image-compress.yml (hunk 1 above)
Comment (wolf-yuan-6115) [reply .github/workflows/image-compress.yml:25]: Fine

**Comment by wolf-yuan-6115** [comment]
MERGE MERGE

**Review by Gary50613 — APPROVED** [review]
LTGM this is really cool!

**Pull request merged**